python/tskit/trees.py

```diff
@@ -4764,7 +4765,7 @@ def genotype_matrix(
 
         :param bool isolated_as_missing: If True, the allele assigned to
             missing samples (i.e., isolated samples without mutations) is
-            the ``missing_data_character``. If False, missing samples will be
+            :data:`.MISSING_DATA` (-1). If False, missing samples will be
```

Comment:
Is this right? Do we assign the *allele* (which is a string) to be "-1"? Surely not: it's the *genotype* that is -1, and the allele is some other thing, perhaps "N"?

Reply:
This is for `genotype_matrix(` though right?

Reply:
Ah, good point. Then the word "allele" is a bit confusing, I think? Perhaps "If True, the genotype value assigned to missing samples (i.e., isolated samples without mutations) is :data:`.MISSING_DATA` (-1). If False, missing samples will be assigned the genotype value corresponding to the ancestral state." (I don't think we require the ancestral state to be always index 0 any more, do we?)

Reply:
Yes, I spotted this error as I was going through. I've updated the text a bit (and also copied to variants, where the same text was used)